website/components/card-list/index.tsx

```diff
@@ -0,0 +1,53 @@
+import s from './style.module.css'
+
+interface Card {
+  image: {
+    src: string
+    alt: string
+  }
+  description: string
+  cta: {
+    url: string
+    text: string
+  }
+  type: string
+}
+
+interface CardListProps {
+  title: string
+  cards: Card[]
+}
+
+export default function CardList({ title, cards }: CardListProps) {
+  return (
+    <>
```

Comment:
Having a component returning multiple top-level objects will cause most of your consumers to have to wrap your component in a div (which is some complexity we could avoid).

I would probably prefer wrapping this component in a div itself, and potentially (if needed) accept a className to this component, and pass it to that div.

Reply:
I agree and like the className idea idea